drivers/google/google_test.go

```diff
@@ -0,0 +1,153 @@
+package google
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/codegangsta/cli"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestDriverName(t *testing.T) {
+	driverName := newDriver().DriverName()
+
+	assert.Equal(t, "google", driverName)
+}
+
+func TestFailWithoutProject(t *testing.T) {
+	_, err := create([]string{"test-name"})
+
+	assert.EqualError(t, err, "Please specify the Google Cloud Project name using the option --google-project.")
+}
+
+func TestParseFlags(t *testing.T) {
+	driver, err := create([]string{
```

Comment:
I think this would be a lot better suited by testing `SetConfigFromFlags` directly.  To do so, just pass in a custom version of the `DriverOptions` interface (use `DriverOptionsMock` from `libmachine/hosttest` if you can and if you run into circular import issues go ahead and move it to a separate module).